### ISSUE TITLE
local projects without a project.yaml file can fail to find the project root

### ISSUE THREAD

**Issue by xtfer** [platformsh/platformsh-cli#440]
I had a recurring issue on a project supposedly compatible with CLI 3.x where a number of commands including local:clean would return RootNotFoundException, despite being in the project root.

I eventually solved this by manually adding an empty project.yaml file to .platform/local, to force \Platformsh\Cli\Local\LocalProject::getProjectRoot to return true, but I'm guessing this is a workaround because the following code in that was returning false BEFORE it had a chance to write the configuration. The project is attached to a bitbucket repo, if that makes any difference.

` 
$gitUrl = $this->getGitRemoteUrl($dir);
if (!$gitUrl || !($projectId = $this->getProjectId($gitUrl))) {
  return false;
}`

**Comment by pjcdawkins** [comment]
I'm not sure I understand the bug - there should always be a `project.yaml` file after the CLI has run at least once in a project - anyway, this may or may not be affected by this fix:
https://github.com/platformsh/platformsh-cli/commit/213ae028

**Comment by xtfer** [comment]
I think this was on an upgrade - the file was definitely missing, but as all my repos are now up to date I can't replicate it. Feel free to close.

**Comment by pjcdawkins** [comment]
OK, thanks for the update

**Comment by josephdpurcell** [comment]
I ran into this same issue. I see the referenced fix, https://github.com/platformsh/platformsh-cli/commit/213ae028, was released several days ago in https://github.com/platformsh/platformsh-cli/releases/tag/v3.25.1.

After `platform self:update` and re-running `platform local:build`, the file is still not created.

Perhaps this is because of how I'm setting up my project. Instead of running `platform get` I am cloning from a gitlab repo then running `platform local:build`. Assuming platform get is what creates the project.yaml file, is there an alternative command that will create it as well on an existing repo?
